embulk-output-db2/build.gradle

```diff
@@ -4,5 +4,5 @@ dependencies {
     compile project(':embulk-output-jdbc')
 
     testCompile project(':embulk-output-jdbc').sourceSets.test.output
-    testCompile files('test_jdbc_driver/db2jcc4.jar')
+    testCompile "com.ibm.db2.jcc:db2jcc:db2jcc4"
```

Comment:
Nice to leave a comment the same with `embulk-input-jdbc`.

```
    // The driver is available on Maven Central since Nov 30, 2018.
    // https://repo1.maven.org/maven2/com/ibm/db2/jcc/db2jcc/db2jcc4/
    //
    // Its license is, however, not a common open-sourcing one.
    // So, we use it only in tests. The plugin artifact does not depend on nor contain it.
```